features/src/main/feature/camel-features.xml

```diff
@@ -62,9 +62,10 @@
 
     <feature name="jaxb-runtime" version="${jaxb-core-version}">
         <feature version="[4,5)">jakarta-xml-bind</feature>
-        <bundle dependency="true">mvn:com.sun.xml.bind/jaxb-core/${jaxb-core-version}</bundle>
-        <bundle dependency="true">mvn:com.sun.xml.bind/jaxb-impl/${jaxb-impl-version}</bundle>
-        <bundle dependency="true">mvn:com.sun.xml.bind/jaxb-osgi/${jaxb-osgi-version}</bundle>
+        <bundle dependency="true">mvn:com.sun.istack/istack-commons-runtime/${auto-detect-version}</bundle>
+        <bundle dependency="true">mvn:org.glassfish.jaxb/jaxb-core/${glassfish-jaxb-runtime-version}</bundle>
+        <bundle dependency='true'>mvn:org.glassfish.jaxb/jaxb-runtime/${glassfish-jaxb-runtime-version}</bundle>
+        <bundle>mvn:org.glassfish.hk2/osgi-resource-locator/${osgi-resource-locator-version}</bundle>
```

Comment:
`camel-jaxb` refuses to work without `jaxb-runtime`. 
 According to the last item, "Preferred JAX-B implementation", in https://camel.apache.org/manual/camel-4x-upgrade-guide-4_8.html we should use `glassfish`. 
 
Ref to the libraries added: https://central.sonatype.com/artifact/org.glassfish.jaxb/jaxb-core/4.0.5. `osgi-resource-locator` appears to be the equivalent of `jaxb-osgi`.

I'm not sure if I'm still missing something, because in the jaxb test, I see this log:
```
2024-09-30T20:12:43,861 | INFO  | RMI TCP Connection(4)-127.0.0.1 | StaxConverter                    | 70 - camel-xml-jaxp - 4.8.0.SNAPSHOT | Created XMLInputFactory: com.sun.xml.internal.stream.XMLInputFactoryImpl@3af22731. DOMSource/DOMResult may have issues with com.sun.xml.internal.stream.XMLInputFactoryImpl@3af22731. We suggest using **Woodstox.**
```